modules/pam_xauth/pam_xauth.c

```diff
@@ -685,13 +704,17 @@ pam_sm_open_session (pam_handle_t *pamh, int flags UNUSED,
 			goto cleanup;
 		}
 
+		if (debug) {
+			pam_syslog(pamh, LOG_DEBUG, "set environment variable '%s'",
+				   xauthority);
+		}
```

Comment:
What it has to do with NFS and the cookiefile?

Reply:
Just a log for debug, not mandatory. I removed it from the new commits.